lib/prefab/options.rb

```diff
@@ -67,7 +67,7 @@ def initialize(
       @prefab_datasources = prefab_datasources
       @prefab_config_classpath_dir = prefab_config_classpath_dir
       @prefab_config_override_dir = prefab_config_override_dir
-      @defaults_env = defaults_env
+      @prefab_envs = [].push(prefab_envs).flatten(1)
```

Comment:
Take it or leave it, but I think this might be more idiomatic as `Array(prefab_envs).flatten(1)`

For my education, why do we need to specify the level of flattening (`1`) here?

Reply:
This was a SO special. Looks like flatten handles nested things better, but no reason we need to support that. Sounds like `Array(prefab_envs)` is sufficient here